pkg/piperutils/fileUtils_test.go

```diff
@@ -32,12 +32,25 @@ func TestCopy(t *testing.T) {
 	file := filepath.Join(dir, "testFile")
 	err = ioutil.WriteFile(file, []byte{byte(1), byte(2), byte(3)}, 0700)
 	if err != nil {
-		t.Fatal("Failed to create temporary workspace directory")
+		t.Fatal("Failed to create temporary workspace directory")
 	}
-	// clean up tmp dir
+	// clean up tmp dir
 	defer os.RemoveAll(dir)
 
 	result, err := Copy(file, filepath.Join(dir, "testFile2"))
 	assert.NoError(t, err, "Didn't expert error but got one")
 	assert.Equal(t, int64(3), result, "Expected true but got false")
 }
+
+func TestDownload(t *testing.T) {
+	dir, err := ioutil.TempDir("", "dir3")
+	targetFile := filepath.Join(dir, "testfile")
+	// clean up tmp dir
+	defer os.RemoveAll(dir)
+
+	_, err = Download("https://raw.githubusercontent.com/SAP/jenkins-library/master/README.md", targetFile)
```

Comment:
I strongly recommand not to use real resources during an unit test. We should mock the http call IMO. Why not using the [http facilities](https://github.com/SAP/jenkins-library/blob/master/pkg/http/http.go), e.g. here the `Sender` interface, already added to piper. During the test we can provide a mock for that and avoid real http calls.

Having access to real resources means having failing tests when the resources are not availabe, e.g. since the corresponding service is down or since you can't access it (developing in a plain over the ocean).